roxie/ccd/ccdcontext.cpp

```diff
@@ -3898,6 +3901,30 @@ class CRoxieServerContext : public CRoxieContextBase, implements IRoxieServerCon
         clusterNames.pop();
         clusterWidth = -1;
     }
+
+    virtual bool logStartStopError()
```

Comment:
minor: The name of this function sounds like an action, rather than a question.  E.g. canLogStart/okToLogStart/queryLogStart

Reply:
ok, yes.  I can rename it.